Update configs w.r.t gem5v20.1.0.0

tv-reddy opened the pull request.

tv-reddy committed:
Update configs w.r.t gem5v20.1.0.0

tv-reddy committed:
update launch script

powerjg commented on gem5-configs/system/__init__.py:
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2016 Jason Lowe-Power
+# Copyright (c) 2020 The Regents of the University of California.
```
This is fine, since I'm approving it, and I'm "Jason Lowe-Power" :).

However, generally, you can't "remove" anyone's copyright. Instead of removing, we should *add* another copyright line (if needed).

tv-reddy replied:
I am sorry, I don't know much about copyrights, I have pushed these changes to Gerrit, should I resubmit taking this into account?

powerjg replied:
You should probably revert this change on gerrit. If you make significant changes to a file, you should update the copyright to add your own (or UC Regents, in this case).

powerjg reviewed (commented):
LGTM! Just out of curiosity, did you copy these python file directly from gem5-resources?

tv-reddy commented:
> LGTM! Just out of curiosity, did you copy these python file directly from gem5-resources?

Yes, just updated port names  and also added support for ruby system for spec2006 tests

powerjg commented:
Just to further clarify... are the scripts in this repo *exactly the same* as the ones in gem5-resources?

tv-reddy commented:
> Just to further clarify... are the scripts in this repo _exactly the same_ as the ones in gem5-resources?

The config scripts in this PR were taken from gem5-resources and then I updated the master-slave port names with new terminology, added support for the ruby system (initially we didn't have support for ruby in spec2006 config scripts in gem5-resources).
The scripts in this PR are the same this one https://gem5-review.googlesource.com/c/public/gem5-resources/+/34755

powerjg commented:
Perfect! Thanks!

On Fri, Sep 18, 2020 at 10:49 AM Trivikram Reddy <notifications@github.com>
wrote:

> Just to further clarify... are the scripts in this repo *exactly the same*
> as the ones in gem5-resources?
> The config scripts in this PR were taken from gem5-resources and then I
> updated the master-slave port names with new terminology, added support for
> the ruby system (initially we didn't have support for ruby in spec2006
> config scripts in gem5-resources).
> The scripts in this PR are the same this one
> https://gem5-review.googlesource.com/c/public/gem5-resources/+/34755
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/darchr/spec2006-experiments/pull/1#issuecomment-695002062>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA2YHFSMTVIUQSROS2LUUTSGOMTPANCNFSM4RRK4B4Q>
> .
>


-- 
Jason Lowe-Power (he/him/his)
Assistant Professor, Computer Science Department
University of California, Davis
3049 Kemper Hall
https://arch.cs.ucdavis.edu/
